source/camera/basic/capture_hdr_complete_settings.py

```diff
@@ -17,28 +17,59 @@ def _main():
     camera = app.connect_camera()
 
     print("Configuring acquisition settings different for all HDR acquisitions")
-    settings = zivid.Settings(
-        acquisitions=[
-            zivid.Settings.Acquisition(
-                aperture=8.0,
-                exposure_time=datetime.timedelta(microseconds=10000),
-                brightness=1.8,
-                gain=1.0,
-            ),
-            zivid.Settings.Acquisition(
-                aperture=4.0,
-                exposure_time=datetime.timedelta(microseconds=10000),
-                brightness=1.8,
-                gain=1.0,
-            ),
-            zivid.Settings.Acquisition(
-                aperture=4.0,
-                exposure_time=datetime.timedelta(microseconds=40000),
-                brightness=1.8,
-                gain=2.0,
-            ),
-        ],
-    )
+    if (
```

Comment:
In C++ and C# this is a function called getExposures, maybe try to implement the same here.


https://github.com/zivid/zivid-cpp-samples/blob/d092e7bfe41aec7cd7a14281e28a3b04f9c4a383/source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cpp#L15-L35

Reply:
I think this is just a place were we will differ for pythonic vs C++
I do not think this needs to be made into a function in this case.

Reply:
I can make the print statements match though if you want.